tests/configuration/test_strategies.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import brownie
+
+from brownie import (
+    StrategyCurveYCRV,
+    StrategyDForceDAI,
+    StrategyDForceUSDC,
+    StrategyDForceUSDT,
+    StrategyMStableSavingsTUSD,
+    StrategyMStableSavingsUSDC,
+    StrategyVaultTUSD,
+    StrategyVaultUSDC,
+    StrategyYfii,
+)
+
+strategies = [
+    StrategyCurveYCRV,
+    StrategyDForceDAI,
+    StrategyDForceUSDC,
+    StrategyDForceUSDT,
+    StrategyMStableSavingsTUSD,
+    StrategyMStableSavingsUSDC,
+    StrategyVaultTUSD,
+    StrategyVaultUSDC,
+    StrategyYfii,
+]
+
+
+@pytest.mark.parametrize("Strategy", strategies)
+def test_strategy_deployment(gov, controller, Strategy):
+    strategy = gov.deploy(Strategy, controller)
+    # Double check all the deployment variable values
+    assert strategy.governance() == gov
+    assert strategy.controller() == controller
+    assert strategy.getName() == Strategy._name
+
+
+@pytest.mark.parametrize(
+    "getter,setter,val",
+    [
+        ("governance", "setGovernance", None),
+        ("controller", "setController", None),
+        ("strategist", "setStrategist", None),
+        ("fee", "setFee", 100),
+        ("withdrawalFee", "setWithdrawalFee", 100),
+        ("performanceFee", "setPerformanceFee", 1000),
+    ],
+)
+@pytest.mark.parametrize("Strategy", strategies)
+def test_strategy_setParams(accounts, gov, controller, getter, setter, val, Strategy):
```

Comment:
This is the most up-to-date one, I'll probably refactor the others to make it look like this, if it's not too confusing